config/locales/shared/payment_types.en.yml

```diff
@@ -11,3 +11,4 @@ en:
       WRITEOFFSMALL: "Write off (small)"
       WRITEOFFLARGE: "Write off (Form L)"
       REVERSAL: "Reversal"
+      GOVPAY: "Card (Govpay)"
```

Comment:
@d-a-v-e this needs to be checked with the business users - might need to say "Card" just like Worldpay. Let's discuss at standup.

Reply:
I still think this needs to be confirmed, it might need to be just "Card". @timstone your thoughts?

Reply:
One for Rob Bond I think, I can see an argument for having just card and one for having a reference to govpay (as it differentiates between current Worldpay payments)